repo-index.yml

```diff
@@ -1676,6 +1676,28 @@ plugins:
   name: spring-cloud-dataflow-for-pcf
   updated: "2020-03-04T09:47:00Z"
   version: 1.0.2
+- authors:
```

Comment:
**suggestion** Think you may need to run `go run sort/main.go repo-index.yml`  else a test will fail

Reply:
😔 I thought I had put it in the right place. Fixed.